components/post-page/DeletePostForm.tsx

```diff
@@ -30,6 +32,7 @@ export default function DeletePostForm({ post }: { post: Post }) {
         onClick={() => onOpen()}
         colorScheme="red"
         leftIcon={<Icon as={TrashIcon} />}
+        width={router.asPath == "/account" ? "240px" : "fit-content"}
```

Comment:
A prop should definitely be passed to control this instead of doing it this way. Also curious why this needs to be different on the account page?

Reply:
<img width="534" alt="Screen Shot 2022-02-24 at 1 33 08 PM" src="https://user-images.githubusercontent.com/40670885/155585863-c1abdae7-f466-43fc-8e7e-3983377cef90.png">
<img width="535" alt="Screen Shot 2022-02-24 at 1 34 35 PM" src="https://user-images.githubusercontent.com/40670885/155586109-cfffe9bf-8083-4b6b-afe8-44308df5bac2.png">

Sounds good. See the images attached for the size difference.